### PR TITLE
UGENE-7328. Show group size in MSA Editor

### PR DESCRIPTION
**Pull request by yalgaer**

### CODE REVIEW

File: src/plugins/GUITestBase/src/GUITestBasePlugin.cpp [hunk 1]
```diff
@@ -2235,10 +2235,7 @@ void GUITestBasePlugin::registerTests(UGUITestBase *guiTestBase) {
     REGISTER_TEST(GUITest_common_scenarios_msa_editor::test_0017_2);
 
     REGISTER_TEST(GUITest_common_scenarios_msa_editor::test_0018);
-
     REGISTER_TEST(GUITest_common_scenarios_msa_editor::test_0019);
-    REGISTER_TEST(GUITest_common_scenarios_msa_editor::test_0019_1);
```
Comment (DmitriiSukhomlinov) [inline src/plugins/GUITestBase/src/GUITestBasePlugin.cpp:2240]: Why did you remove these tests?

**Review by DmitriiSukhomlinov — APPROVED** [review]

File: src/plugins/GUITestBase/src/GUITestBasePlugin.cpp (hunk 1 above)
Comment (yalgaer) [reply src/plugins/GUITestBase/src/GUITestBasePlugin.cpp:2240]: All 3 tests had 100% identical code. Looks like the original author copy/pasted them to use as a template for new tests, but have never finished it.
